k8s.io/configmap-nginx.yaml

```diff
@@ -39,23 +39,28 @@ data:
         server_name k8s.io;
         listen 443 ssl;
 
-        if ($arg_go-get = "1") {
-          # This is a go-get operation.
-          # Send any file in any repo to static content.
-          rewrite ^/([^/]*)(/.*)?$  /_golang-go-get/$1.html;
-        }
-
-        location /_golang-go-get {
-          # Serve static content.
-          alias /www/golang;
-        }
-
         location /_healthz {
           add_header Content-Type text/plain;
           return 200 'ok';
         }
 
-        location / {
+        location ~* ^/([^_][^/]*)(/.*)?$ {
```

Comment:
Does this interfere with the normal redirect?

Reply:
`~*` is unnecessary, since this isn't case sensitive. Why not `^/([^/]*)(/.*)?$`, which would match all URLs?